src/main/webapp/app/admin/user-management/user-management.component.ts

```diff
@@ -467,15 +507,18 @@ export class UserManagementComponent implements OnInit, OnDestroy {
     /**
      * Deletes a user
      * @param login of the user that should be deleted
+     * @param batch update will be triggered manually
      */
-    deleteUser(login: string) {
+    deleteUser(login: string, batch: boolean) {
```

Comment:
I think you could use a default value here to not have to change every method call.
```suggestion
    deleteUser(login: string, batch = false) {
```